release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -166,6 +166,17 @@ With the addition of DNS support in the OVN-Kubernetes cluster network provider
 
 For containers interacting with SR-IOV virtual functions (VFs) in Data Plane Development Kit (DPDK) mode, the `app-netutil` library now provides the following functions: `GetCPUInfo()`, `GetHugepages()`, and `GetInterfaces()`. For more information, see xref:../networking/hardware_networks/about-sriov.adoc#nw-sriov-app-netutil_about-sriov[DPDK library for use with container applications].
 
+[id="ocp-4-7-networking-k8s-nmstate"]
+==== Kubernetes NMState (Technology Preview)
```

Comment:
Isn't this an Operator? Shouldn't it be Kubernetes Nmstate Operator, with the proper name consistent with upstream usage?

Reply:
Looks like I'm incorrect, it is _Kubernetes NMState Operator_.